ci/generate-markdown/generate-packaging.sh

```diff
@@ -202,9 +202,9 @@ these dependencies.
 - [Fedora 31](#fedora-31)
 - [openSUSE (Tumbleweed)](#opensuse-tumbleweed)
 - [openSUSE (Leap)](#opensuse-leap)
-- [Ubuntu (20.04 - Bionic Beaver)](#ubuntu-2004---focal-fossa)
-- [Ubuntu (18.04 - Bionic Beaver)](#ubuntu-1804---bionic-beaver)
-- [Ubuntu (16.04 - Xenial Xerus)](#ubuntu-1604---xenial-xerus)
+- [Ubuntu (20.04 LTS - Focal Fossa)](#ubuntu-2004---focal-fossa)
```

Comment:
not sure `#ubuntu-2004---focal-fossa` is still working since I add the LTS (which is part of the official name IIRC)

Reply:
Just use chrome extension markdown viewer to retrieve to the potential new tag name

Reply:
You can see the packaging.md file in your own branch, and follow the links there:

https://github.com/Mizux/google-cloud-cpp/blob/patch-1/doc/packaging.md#table-of-contents

and:

https://github.com/Mizux/google-cloud-cpp/blob/patch-1/doc/packaging.md#ubuntu-2004-lts---focal-fossa

Reply:
tested !